src/Doctrine/IdeaExtension.php

```diff
@@ -16,9 +16,11 @@ public function applyToItem(QueryBuilder $queryBuilder, QueryNameGeneratorInterf
         $this->modifyQuery($queryBuilder, $resourceClass);
     }
 
-    public function applyToCollection(QueryBuilder $queryBuilder, QueryNameGeneratorInterface $queryNameGenerator, string $resourceClass, string $operationName = null)
+    public function applyToCollection(QueryBuilder $queryBuilder, QueryNameGeneratorInterface $queryNameGenerator, string $resourceClass, string $operationName = null, array $context = [])
```

Comment:
can you implement `ContextAwareQueryCollectionExtensionInterface` instead of `QueryCollectionExtensionInterface` for using `$context`